app/views/admin/products/index.html.erb

```diff
@@ -23,6 +23,8 @@
       <td>
         <%= link_to "edit", edit_admin_product_path(product), 
           class: "btn btn-outline-secondary btn-sm", id: "edit-product-#{product.id}-button" %>
+        <%= link_to "delete",  admin_product_path(product), method: :delete, 
+          data: { confirm: 'Are you sure?' }, class: "btn btn-outline-danger btn-sm", id: "delete-product-#{product.id}-button" %>
```

Comment:
po co jest Ci to id?

Reply:
podczas testów chciałem miać pewność, że klikam w odpowiedni button 'edit' i 'delete'

Reply:
to lepiej do tego uzyc aria label lub (jak masz diva) custom attrs.

Reply:
dopisuję to do trello ;)